config/ftbquests/quests/chapters/getting_started.snbt

```diff
@@ -170,586 +170,432 @@
 				""
 				"●Craft the Tome with a book to merge them once more. "
 				""
+				"Note: Akashic Tome is a bit broken, and book is covering half of the manuals. This is only visual bug, so book is 100% functional!"

```

Comment:
Quite a few typos here, I'd rather you remove this section tbh.

Reply:
Ah alrighty. (And sorry for typos, was doing in that time few things at once 😅) Gonna remove it then.